src/main/java/Todolist.java

```diff
@@ -0,0 +1,77 @@
+import java.util.Scanner;
+
+public class Todolist {
+
+    public static final String LINE = "    ____________________________________________________________\n";
+    public static final String GREET = "     Hello! I'm Duke\n" +
+            "     Welcome to Tasks!\n" +
+            "     Enter \"todo 'task-name'\" to add a task.\n" +
+            "     Enter \"deadline 'task-name' /by 'deadline'\" to add a task with a deadline.\n" +
+            "     Enter \"event 'task-name' /from 'start-date' /to 'end-date'\" to add a task with start and end dates.\n" +
+            "     Enter \"mark 'task-index'\" to mark a task as done.\n" +
+            "     Enter \"unmark 'task-index'\" to mark a task as not done yet.\n" +
+            "     Enter \"list\" to obtain a list of all your tasks!.\n";
+    public static final String BYE = "     Bye. Hope to see you again soon!\n";
+    public static final String NOT_DONE = "    OK :(, I've marked this task as not done yet: \n    ";
+    public static final String DONE = "    Nice! I've marked this task as done: \n    ";
+    public static final String ERROR = "    Something went wrong :( Check your input and try again! \n";
```

Comment:
I like the naming of constant variables with all capital letters and using underscores to separate words.